src/content/docs/apm/agents/python-agent/installation/python-agent-admin-script-advanced-usage.mdx

```diff
@@ -25,26 +25,28 @@ This document gives more in-depth detail about these uses of the admin script:
 
 To execute subcommands, provide the name of the subcommand as the first argument to the `newrelic-admin` script. Then add any options after the name of the subcommand. For example:
 
-```
+```shell
 newrelic-admin help
 ```
 
 ## help &lt;command> [#help]
 
 When executed with no further arguments, this provides a listing of all accepted subcommands:
 
-```
-$ newrelic-admin help
-    Usage: newrelic-admin command [options]
+```shell
+newrelic-admin help
+[output]  Usage: newrelic-admin command [options]
 
-    Type 'newrelic-admin help <command>' for help on any of the commands listed in this document.
+[output]  Type 'newrelic-admin help <command>' for help on any of the commands listed in this document.
+```
 
 To view additional details about a subcommand, run the help command with the subcommand and any of its options. For example:
 
-    $ newrelic-admin help generate-config
-    Usage: newrelic-admin generate-config license_key [output_file]
+```shell
+newrelic-admin help generate-config
+[output]  Usage: newrelic-admin generate-config license_key [output_file]
```

Comment:
@brnhensley I'm not quite sure I understand the syntax here. The literal for `[output]` isn't getting rendered at the beginning of line 47. When I run this locally, I get this output:
```
Usage: newrelic-admin generate-config license_key [output_file]

Generates a sample agent configuration file for <license_key>.
```

Reply:
It's probably extra confusing because the `[output]` is just for the terminal output formatting, but the output itself contains  `[output_file]` which looks so similar. `[output]`  is not supposed to render as text here

Reply:
@brnhensley thanks for clarifying that `[output]` is part of the formatting!